src/foam/util/StringUtil.java

```diff
@@ -90,9 +90,13 @@ public static String labelize(String s) {
     var sb = new StringBuilder();
     char current = 0;
     for ( var c : s.toCharArray() ) {
-      if ( current == 0 ) current = Character.toUpperCase(c);
-      else {
-        if ( ! Character.isUpperCase(current) && Character.isUpperCase(c) ) {
+      if ( current == 0 ) {
```

Comment:
Does the JS labelize function already do this?

Reply:
Need to use it on the server side, the JS version didn't get called.